DateTimePicker/DateTimePicker.module.less

```diff
@@ -34,7 +35,6 @@
 .applySkins({
 	.dateTimePicker {
 		font-size: @agate-picker-font-size;
-		width: @agate-datetimepicker-width;
```

Comment:
I think that the approach to fix the bug and most of the code looks good to me. But if reducing resolution, the DateTimePicker layout is broken. It might be related with this line. Do you have any idea to fix this issue?

![image](https://user-images.githubusercontent.com/4239873/95283209-2d1e2900-0896-11eb-8b28-298a719517bc.png)

Reply:
I guess it is just too much content in the screen, for that particular resolution. The only thing that seems to fix it is to make the Picker padding 6px instead of 12px